templates/quickstart-jira-dc.template.yaml

```diff
@@ -842,60 +813,58 @@ Mappings:
     z1d.12xlarge:
       Arch: HVM64
       Jvmheap: 12288m
+
   AWSRegionArch2AMI:
     ap-northeast-1:
-      HVM64: ami-0cf893fd749668192
+      HVM64: ami-00d101850e971728d
       HVMG2: NOT_SUPPORTED
     ap-northeast-2:
-      HVM64: ami-0cce7f639d43e2f99
+      HVM64: ami-08ab3f7e72215fe91
```

Comment:
Nice, nice - https://aws.amazon.com/amazon-linux-2/release-notes/

We should flash out some basic release notes for the change, do you plan to do this @tarka ?

Reply:
I can do. I'm not sure where they would go. Do we have release notes for the Quickstarts?